tools/serverpod_cli/lib/src/analyzer/entities/entity_parser/entity_parser.dart

```diff
@@ -34,6 +34,8 @@ class EntityParser {
     if (className is! String) return null;
 
     var tableName = _parseTableName(documentContents);
+    var viewName = _parseViewName(documentContents);
+    var queryView = _parseQueryName(documentContents);
```

Comment:
Suggestion: rename to `viewQuery` and `_parseViewQuery`.

I think that name better reflects what we are parsing.